.rosinstall

```diff
@@ -3,6 +3,8 @@
 # IF YOU WANT TO CHANGE THE ROS ENVIRONMENT VARIABLES
 # USE THE rosinstall TOOL INSTEAD.
 # IF YOU CHANGE IT, USE rosinstall FOR THE CHANGES TO TAKE EFFECT
-- git: {local-name: src/uwsim, 
-        uri: 'https://github.com/uji-ros-pkg/underwater_simulation.git', 
-        version: kinetic-devel}
+- setup-file: {local-name: /opt/ros/kinetic/setup.sh}
+- git: {local-name: /usr/share/ros/uwsim, uri: 'https://github.com/uji-ros-pkg/underwater_simulation.git',
+    version: kinetic-devel}
```

Comment:
CI doesn't check this, so can you confirm that the sim still runs fine after you delete `install`, `devel`, `install-isolated`, and `devel-isolated` (if present) in `Jormungandr` (just to make sure we're not using anything from old install locations)?

Reply:
I did a clean install with .rosinstall on another computer, so everything should be working.